mockgen/mockgen.go

```diff
@@ -365,10 +366,13 @@ func (g *generator) generateOngoingVerificationGetAllCapturedArguments(ongoingVe
 					p("}")
 				break
 			} else {
+				// explicitly validate the length of the params slice to avoid out of bounds code smells
+				g.p("if %v < len(_params) {", i)
```

Comment:
Could you please change this to:
```
g.p("if len(_params) > %v {", i)
```
Logically, it's the same, but it reads more naturally.

Reply:
Agreed, updated in https://github.com/petergtz/pegomock/pull/126/commits/54c3d6ebe5c03819d28b61444c6f80e9a6c1a061